apps/testingaccessibility/src/data/migrate-bulk-coupon-purchases-relation.ts

```diff
@@ -0,0 +1,79 @@
+// Running this script:
+//
+// Run the following commands from 'apps/testingaccessibility'.
+//
+// First, open a connection to the planetscale database and branch:
+//
+// ```
+// pscale connect testing-accessbility <branch-name> --port 3309 --host "::1"
+// ```
+//
+// Execute the data migration:
+//
+// ```
+// npx ts-node --skipProject src/data/migrate-bulk-coupon-purchases-relation.ts
+// ```
+
+import {prisma} from '@skillrecordings/database'
+
+const backfillBulkCouponIdForBulkPurchases = async () => {
```

Comment:
This one-off script is Step 3 ([EaC](https://www.prisma.io/dataguide/types/relational/expand-and-contract-pattern)), migrate existing data from the current schema to the new schema. This script finds all the one-to-one Bulk Purchase <-> Bulk Coupon records and backfills the `Purchase.bulkCouponId` column with those Bulk Coupon `id` values.